samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/model/enum_test.dart

```diff
@@ -110,11 +112,12 @@ class EnumTestEnumInteger extends EnumClass {
 }
 
 
+
 class EnumTestEnumNumber extends EnumClass {
 
-  @BuiltValueEnumConst(wireName: '1.1')
+  @BuiltValueEnumConst()
   static const EnumTestEnumNumber 11_ = _$11_;
-  @BuiltValueEnumConst(wireName: '-1.2')
```

Comment:
See above comment in template. Sadly `wireNumber: ` doesn't support num/double. Probably need to add a custom serializer for this at some point.